src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -33,6 +33,13 @@ Released 2023-Jan-30
   environment variables) used by GenevaTraceExporter in Linux.
   ([#925](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/925))
 
+* Changed the behavior of Unix domain socket connection at startup. Before this
```

Comment:
After merges with main it was moved to relesed section,. It should be under unreleased.